nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -76,6 +76,35 @@ NOTE: For security purposes, when no security configuration is provided NiFi wil
 
 See the <<system_properties>> section of this guide for more information about configuring NiFi repositories and configuration files.
 
+== Build a Custom Distribution
+
+The binary build of Apache NiFi that is provided by the Apache mirrors does not contain every NAR file that is part of the official release. This is due to size constraints imposed by the mirrors to reduce the expenses associated with hosting such a large project. The following Maven profiles are available to administrators that want to simplify the process of building in-house distributions of Apache NiFi that contain the additional functionality that is normally excluded from the official binary releases.
+
+|==================================================================================================================================================
+| Package                                         | Maven Profile       | Description
+| Apache Accumulo Bundle | include-accumulo   | Adds support for https://accumulo.apache.org:[Apache Accumulo].
+| Apache Atlas Bundle | include-atlas | Adds support for the Apache Atlas data governance tool. The functionality from this bundle is based around reporting tasks that integrate with https://atlas.apache.org/#/:[Apache Atlas].
```

Comment:
URL goes to "Page Not Found".  I think the colon at the end should be removed:

https://atlas.apache.org/#/

Reply:
All the extraneous colons should be removed from the URLs in the table.